website/src/documentation/transforms/java/index.md

```diff
@@ -58,6 +58,7 @@ limitations under the License.
   <tr><td><a href="{{ site.baseurl }}/documentation/transforms/java/aggregation/groupbykey">GroupByKey</a></td><td>Takes a keyed collection of elements and produces a collection where each element 
   consists of a key and all values associated with that key.</td></tr>
   <tr><td><a href="{{ site.baseurl }}/documentation/transforms/java/aggregation/groupintobatches">GroupIntoBatches</a></td><td>Batches values associated with keys into <code>Iterable</code> batches of some size. Each batch contains elements associated with a specific key.</td></tr>
+  <tr><td><a href="{{ site.baseurl }}/documentation/transforms/java/aggregation/hllcount">HllCount</a></td><td>Estimate the number of distinct elements and creating sketches using the HyperLogLog++ algorithm.</td></tr>
```

Comment:
Would also leave a comment at the ApproximateUnique entry, something like "See also HllCount". Or should we leave that to the person who might look into marking that aggregation as deprecated?

Reply:
Added a reference to `HllCount` in `ApproximateUnique` catalog.